karma.conf.js

```diff
@@ -73,6 +73,7 @@ module.exports = function(config) {
 
 		webpack: {
 			watch: true,
+			devTool: 'eval',
```

Comment:
Threw this in here since we don't need sourcemaps and it _might_ speed things up a little bit.